app/views/games/show.html.erb

```diff
@@ -19,7 +19,13 @@
       <% 8.times do %>
         <% file = file + 1 %>
         <div class="square">
-          <p><%= "#{file}, #{rank}" %></p>
+          <div class=<%= "file#{file}rank#{rank}" %>>
+            <% current_game.pieces.each do |piece| %>
+              <% if piece.rank == rank && piece.file == file %>
+                <%= "#{piece.color} #{piece.subclasses}" %>
```

Comment:
`#{piece.subclasses}` this won't be valid since .subclasses is not a valid Piece model attribute and Rails will most likely throw an error or not show anything at all.

I recommend changing this to `#{piece.type}`  since that is a valid column in the Pieces table